pkg/apis/application/v1alpha1/types.go

```diff
@@ -1342,7 +1353,8 @@ func (p *AppProject) GetRoleByName(name string) (*ProjectRole, int, error) {
 }
 
 // GetJWTToken looks up the index of a JWTToken in a project by id (new token), if not then by the issue at time (old token)
-func (p *AppProject) GetJWTToken(roleName string, issuedAt int64, id string) (*JWTToken, int, error) {
+func (p *AppProject) GetJWTTokenFromSpec(roleName string, issuedAt int64, id string) (*JWTToken, int, error) {
```

Comment:
The `GetJWTTokenFromSpec` can be made private now. Please rename to `getJWTTokenFromSpec`